plugins/completion/cmp/sources/codeium-nvim.nix

```diff
@@ -1,85 +1,96 @@
 {
   lib,
-  helpers,
-  config,
   pkgs,
   ...
 }:
-with lib;
 let
-  cfg = config.plugins.codeium-nvim;
+  inherit (lib.nixvim) defaultNullOpts;
 in
-{
-  meta.maintainers = [ maintainers.GaetanLepage ];
+lib.nixvim.neovim-plugin.mkNeovimPlugin {
+  name = "codeium-nvim";
+  originalName = "codeium.nvim";
+  luaName = "codeium";
+  defaultPackage = pkgs.vimPlugins.codeium-nvim;
```

Comment:
```suggestion
```

Reply:
If the package attrname was different to `name`, we could specify it as a string to `package`.